core/src/main/java/org/infinispan/globalstate/impl/GlobalConfigurationManagerImpl.java

```diff
@@ -172,11 +202,15 @@ void start() {
 
    CompletableFuture<Void> createCacheLocally(String name, CacheState state) {
       log.debugf("Create cache %s", name);
-      ConfigurationBuilderHolder builderHolder = parserRegistry.parse(state.getConfiguration());
-      Configuration configuration = builderHolder.getNamedConfigurationBuilders().get(name).build();
+      Configuration configuration = buildConfiguration(name, state);
       return localConfigurationManager.createCache(name, state.getTemplate(), configuration, state.getFlags());
    }
 
+   private Configuration buildConfiguration(String name, CacheState state) {
+      ConfigurationBuilderHolder builderHolder = parserRegistry.parse(state.getConfiguration());
+      return builderHolder.getNamedConfigurationBuilders().get(name).build();
```

Comment:
I'm not sure if this is ok. Some attributes are validated against the global configuration, and they may not be valid with the current global configuration.

Reply:
I now build against the global configuration